petermwash/JavaDevsNairobi
# #158792524 #159051218 design mockups and color palette for the app.

#### What does this PR do?
- Design mockups (wireframes).
- Determine the color palette for the app.

#### Description of tasks to be completed
- Design wireframes for the app, add them in a wireframes directory in the root directory and link them to the README.
- Determine the color palette for them and update the `color.xml` file with the new palettes.

#### How should this be manually tested?
- Navigate to the `wireframes` folder in the root directory and click on each mockup to view them.
- You should be able to see mockups as shown below.

Main screen
<img width="332" alt="main screen" src="https://user-images.githubusercontent.com/27728464/46999501-87180800-d12e-11e8-9f4e-f1d89b8f5742.png">

Detail screen
<img width="347" alt="details screen" src="https://user-images.githubusercontent.com/27728464/46999522-94cd8d80-d12e-11e8-9d57-8ce4ef3e3afd.png">

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#158792524](https://www.pivotaltracker.com/story/show/158792524)
[#159051218](https://www.pivotaltracker.com/story/show/159051218)
